<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Form</class>
 <widget class="QWidget" name="Form">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>585</width>
    <height>873</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout" stretch="0">
   <property name="leftMargin">
    <number>0</number>
   </property>
   <property name="topMargin">
    <number>0</number>
   </property>
   <property name="rightMargin">
    <number>0</number>
   </property>
   <property name="bottomMargin">
    <number>0</number>
   </property>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_3" stretch="0">
     <property name="spacing">
      <number>0</number>
     </property>
     <item>
      <widget class="QWidget" name="widget" native="true">
       <property name="maximumSize">
        <size>
         <width>16777215</width>
         <height>16777215</height>
        </size>
       </property>
       <property name="styleSheet">
        <string notr="true">.QWidget{background-color: rgb(25,25, 40);}

ComboBox{
border-radius: 10px;
      padding: 0 8px;}
QLineEdit{
border-radius: 10px;
      padding: 0 8px;
font: 12pt &quot;Verdana&quot;;
}
#QComboBox{
font: 12pt &quot;Verdana&quot;;
}
#cancelBtn:hover {
background-color:red;
}

#confirmBtn:hover{
background-color:rgb(105, 95, 20);
}

QLabel{
color: rgb(231, 231, 231);
  font: 11pt &quot;Verdana&quot;;
}</string>
       </property>
       <widget class="QFrame" name="frame">
        <property name="geometry">
         <rect>
          <x>10</x>
          <y>20</y>
          <width>16</width>
          <height>16</height>
         </rect>
        </property>
        <property name="frameShape">
         <enum>QFrame::StyledPanel</enum>
        </property>
        <property name="frameShadow">
         <enum>QFrame::Raised</enum>
        </property>
       </widget>
       <widget class="QPushButton" name="confirmBtn">
        <property name="geometry">
         <rect>
          <x>380</x>
          <y>740</y>
          <width>140</width>
          <height>50</height>
         </rect>
        </property>
        <property name="styleSheet">
         <string notr="true">color: rgb(231, 231, 231);
font: 17pt &quot;Verdana&quot;;
border: 2px solid orange;
padding: 5px;
border-radius: 3px;
opacity: 200;
</string>
        </property>
        <property name="text">
         <string>Confirm</string>
        </property>
       </widget>
       <widget class="QLabel" name="picpUpLbl">
        <property name="geometry">
         <rect>
          <x>50</x>
          <y>650</y>
          <width>120</width>
          <height>40</height>
         </rect>
        </property>
        <property name="font">
         <font>
          <family>Verdana</family>
          <pointsize>11</pointsize>
          <weight>50</weight>
          <italic>false</italic>
          <bold>false</bold>
         </font>
        </property>
        <property name="text">
         <string>Pick-up Point:</string>
        </property>
       </widget>
       <widget class="QLabel" name="sContactNoLbl">
        <property name="geometry">
         <rect>
          <x>50</x>
          <y>230</y>
          <width>150</width>
          <height>40</height>
         </rect>
        </property>
        <property name="font">
         <font>
          <family>Verdana</family>
          <pointsize>11</pointsize>
          <weight>50</weight>
          <italic>false</italic>
          <bold>false</bold>
         </font>
        </property>
        <property name="text">
         <string>Sender Contact No:</string>
        </property>
       </widget>
       <widget class="QLineEdit" name="pickupInput">
        <property name="geometry">
         <rect>
          <x>220</x>
          <y>650</y>
          <width>300</width>
          <height>40</height>
         </rect>
        </property>
        <property name="font">
         <font>
          <family>Verdana</family>
          <pointsize>12</pointsize>
          <weight>50</weight>
          <italic>false</italic>
          <bold>false</bold>
         </font>
        </property>
       </widget>
       <widget class="QLineEdit" name="destinationInput">
        <property name="geometry">
         <rect>
          <x>220</x>
          <y>590</y>
          <width>300</width>
          <height>40</height>
         </rect>
        </property>
        <property name="font">
         <font>
          <family>Verdana</family>
          <pointsize>12</pointsize>
          <weight>50</weight>
          <italic>false</italic>
          <bold>false</bold>
         </font>
        </property>
       </widget>
       <widget class="QPushButton" name="cancelBtn">
        <property name="geometry">
         <rect>
          <x>220</x>
          <y>740</y>
          <width>140</width>
          <height>50</height>
         </rect>
        </property>
        <property name="styleSheet">
         <string notr="true">color: rgb(231, 231, 231);
font: 17pt &quot;Verdana&quot;;
border: 2px solid orange;
padding: 5px;
border-radius: 3px;
opacity: 200;
</string>
        </property>
        <property name="text">
         <string>Cancel</string>
        </property>
       </widget>
       <widget class="QComboBox" name="parcelWeight">
        <property name="geometry">
         <rect>
          <x>220</x>
          <y>410</y>
          <width>300</width>
          <height>41</height>
         </rect>
        </property>
        <property name="styleSheet">
         <string notr="true">border-radius: 10px;
      padding: 0 8px;
font: 15pt &quot;Verdana&quot;;</string>
        </property>
        <item>
         <property name="text">
          <string>Select Weight</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>0.1KG - 1KG</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>1KG - 3KG</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>3KG+</string>
         </property>
        </item>
       </widget>
       <widget class="QLabel" name="destinationLbl">
        <property name="geometry">
         <rect>
          <x>50</x>
          <y>590</y>
          <width>150</width>
          <height>40</height>
         </rect>
        </property>
        <property name="font">
         <font>
          <family>Verdana</family>
          <pointsize>11</pointsize>
          <weight>50</weight>
          <italic>false</italic>
          <bold>false</bold>
         </font>
        </property>
        <property name="text">
         <string>Destination:</string>
        </property>
       </widget>
       <widget class="QLabel" name="sNameLbl">
        <property name="geometry">
         <rect>
          <x>50</x>
          <y>170</y>
          <width>111</width>
          <height>40</height>
         </rect>
        </property>
        <property name="font">
         <font>
          <family>Verdana</family>
          <pointsize>11</pointsize>
          <weight>50</weight>
          <italic>false</italic>
          <bold>false</bold>
         </font>
        </property>
        <property name="text">
         <string>Sender Name:</string>
        </property>
       </widget>
       <widget class="QComboBox" name="courier">
        <property name="geometry">
         <rect>
          <x>220</x>
          <y>530</y>
          <width>300</width>
          <height>40</height>
         </rect>
        </property>
        <property name="styleSheet">
         <string notr="true">border-radius: 10px;
      padding: 0 8px;
font: 15pt &quot;Verdana&quot;;</string>
        </property>
        <item>
         <property name="text">
          <string>Select Courier</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>J&amp;T EXPRESS</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>JRS EXPRESS</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>NINJA VAN</string>
         </property>
        </item>
       </widget>
       <widget class="QLineEdit" name="sNameInput">
        <property name="geometry">
         <rect>
          <x>220</x>
          <y>170</y>
          <width>300</width>
          <height>41</height>
         </rect>
        </property>
        <property name="font">
         <font>
          <family>Verdana</family>
          <pointsize>12</pointsize>
          <weight>50</weight>
          <italic>false</italic>
          <bold>false</bold>
         </font>
        </property>
       </widget>
       <widget class="QSpinBox" name="parcelQuantity">
        <property name="geometry">
         <rect>
          <x>220</x>
          <y>470</y>
          <width>300</width>
          <height>40</height>
         </rect>
        </property>
        <property name="font">
         <font>
          <family>Verdana</family>
          <pointsize>15</pointsize>
          <weight>50</weight>
          <italic>false</italic>
          <bold>false</bold>
         </font>
        </property>
        <property name="styleSheet">
         <string notr="true">border-radius: 10px;
      padding: 0 8px;
font: 15pt &quot;Verdana&quot;;</string>
        </property>
       </widget>
       <widget class="QLabel" name="courierLbl">
        <property name="geometry">
         <rect>
          <x>50</x>
          <y>530</y>
          <width>160</width>
          <height>40</height>
         </rect>
        </property>
        <property name="font">
         <font>
          <family>Verdana</family>
          <pointsize>11</pointsize>
          <weight>50</weight>
          <italic>false</italic>
          <bold>false</bold>
         </font>
        </property>
        <property name="text">
         <string>Courier: </string>
        </property>
       </widget>
       <widget class="QLabel" name="rNameLbl">
        <property name="geometry">
         <rect>
          <x>50</x>
          <y>290</y>
          <width>120</width>
          <height>40</height>
         </rect>
        </property>
        <property name="font">
         <font>
          <family>Verdana</family>
          <pointsize>11</pointsize>
          <weight>50</weight>
          <italic>false</italic>
          <bold>false</bold>
         </font>
        </property>
        <property name="text">
         <string>Receiver Name:</string>
        </property>
       </widget>
       <widget class="QLabel" name="parcelQunatityLbl">
        <property name="geometry">
         <rect>
          <x>50</x>
          <y>470</y>
          <width>120</width>
          <height>40</height>
         </rect>
        </property>
        <property name="font">
         <font>
          <family>Verdana</family>
          <pointsize>11</pointsize>
          <weight>50</weight>
          <italic>false</italic>
          <bold>false</bold>
         </font>
        </property>
        <property name="text">
         <string>Parcel Quantity</string>
        </property>
       </widget>
       <widget class="QLineEdit" name="rNameInput">
        <property name="geometry">
         <rect>
          <x>220</x>
          <y>290</y>
          <width>300</width>
          <height>41</height>
         </rect>
        </property>
        <property name="font">
         <font>
          <family>Verdana</family>
          <pointsize>12</pointsize>
          <weight>50</weight>
          <italic>false</italic>
          <bold>false</bold>
         </font>
        </property>
       </widget>
       <widget class="QLineEdit" name="rContactInput">
        <property name="geometry">
         <rect>
          <x>220</x>
          <y>350</y>
          <width>300</width>
          <height>41</height>
         </rect>
        </property>
        <property name="font">
         <font>
          <family>Verdana</family>
          <pointsize>12</pointsize>
          <weight>50</weight>
          <italic>false</italic>
          <bold>false</bold>
         </font>
        </property>
       </widget>
       <widget class="QLineEdit" name="sContactInput">
        <property name="geometry">
         <rect>
          <x>220</x>
          <y>230</y>
          <width>300</width>
          <height>41</height>
         </rect>
        </property>
        <property name="font">
         <font>
          <family>Verdana</family>
          <pointsize>12</pointsize>
          <weight>50</weight>
          <italic>false</italic>
          <bold>false</bold>
         </font>
        </property>
       </widget>
       <widget class="QLabel" name="rContactNoLbl">
        <property name="geometry">
         <rect>
          <x>50</x>
          <y>350</y>
          <width>160</width>
          <height>40</height>
         </rect>
        </property>
        <property name="font">
         <font>
          <family>Verdana</family>
          <pointsize>11</pointsize>
          <weight>50</weight>
          <italic>false</italic>
          <bold>false</bold>
         </font>
        </property>
        <property name="text">
         <string>Receiver Contact No:</string>
        </property>
       </widget>
       <widget class="QLabel" name="parcelTypeLbl">
        <property name="geometry">
         <rect>
          <x>50</x>
          <y>410</y>
          <width>150</width>
          <height>40</height>
         </rect>
        </property>
        <property name="font">
         <font>
          <family>Verdana</family>
          <pointsize>11</pointsize>
          <weight>50</weight>
          <italic>false</italic>
          <bold>false</bold>
         </font>
        </property>
        <property name="text">
         <string>Parcel Type:</string>
        </property>
       </widget>
       <widget class="QLabel" name="label">
        <property name="geometry">
         <rect>
          <x>0</x>
          <y>0</y>
          <width>591</width>
          <height>61</height>
         </rect>
        </property>
        <property name="styleSheet">
         <string notr="true">background-color: orange;
padding-left: 140px;
font: 22pt &quot;Courier New&quot;;
font-weight:bold;</string>
        </property>
        <property name="text">
         <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; color:#191928;&quot;&gt;J4 Courier Center&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
        </property>
       </widget>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
